libkbfs/mdserver_remote.go

```diff
@@ -351,6 +351,10 @@ func (md *MDServerRemote) OnDisconnected(ctx context.Context,
 		md.serverOffset = 0
 	}()
 
+	md.authenticatedMtx.Lock()
```

Comment:
Don't you usually do the following approach to insure against panics? Or do you only do that when there's a reasonable chance that the line can panic?
```go
func() {
    md.authenticatedMtx.Lock()
    defer md.authenticatedMtx.Unlock()
    md.isAuthenticated = false
}()
```